walrus/FileConfig.py

```diff
@@ -0,0 +1,20 @@
+class FileConfig:
+    def __init__(self, file_path):
+        self._country_capital = {}
+        self._set_capital(file_path)
+
+    def _set_capital(self, file_path):
+        with open(file_path, 'r') as file:
+            data = file.readlines()
+
+            header = data[0].split(",")
+            country_col = header.index('CountryName')
+            capital_col = header.index('CapitalName')
+
+            self._country_capital = {
```

Comment:
The same operation is happening twice here -`line.split(",")`

Reply:
Assigned line.split(",") to a list of list to prevent it from occurring twice.